pkg/utils/validator/clusterresourceplacement_test.go

```diff
@@ -175,6 +175,12 @@ func Test_validateClusterResourcePlacementAlpha(t *testing.T) {
 func Test_validateClusterResourcePlacement(t *testing.T) {
 	unavailablePeriodSeconds := -10
 	var numberOfClusters int32 = 1
+	resourceSelector := placementv1beta1.ClusterResourceSelector{
```

Comment:
This one function test all is not easy to follow and maintain. There are going to be too many cases. Why not split them into UT? This is more like e2e.